changelog/pending/20240919--engine--add-input-keys-from-diff-during-import-step-to-warning-and-error-message-fixing-15606.yaml

```
@@ -0,0 +1,4 @@
+changes:
+- type: fix
+  scope: engine
+  description: "Add input keys from diff during import step to warning and error message, fixing #15606."
```

```suggestion
  description: Add input keys from diff during import step to warning and error message
```

We automatically link back to the PR, so mentioning the issue is not necessary.

---

updated